packages/quick_actions/quick_actions_android/example/pubspec.yaml

```diff
@@ -26,3 +26,8 @@ dev_dependencies:
 
 flutter:
   uses-material-design: true
+
+# FOR TESTING AND INITIAL REVIEW ONLY. DO NOT MERGE.
```

Comment:
Please revert all of the `quick_actions_android` override entries since the package won't need any changes.